src/cli/storage.ts

```diff
@@ -0,0 +1,31 @@
+import { getItemSync, setItemSync, init } from "node-persist";
+import { TelemetryState } from "./telemetry";
+
+
+
+export class Storage {
```

Comment:
What's the point of this class again?  If you want the functions in a namespace you can always do `import * as Storage from "Storage"`.